Slim/Player/Protocols/HTTP.pm

```diff
@@ -638,12 +638,12 @@ sub sysread {
 	my $readLength;
 
 	# do not read if we are building-up too much processed audio
-	if (${*$self}{'audio_buildup'} > $chunkSize) {
-		${*$self}{'audio_buildup'} = ${*$self}{'audio_process'}->(${*$self}{'audio_stash'}, $_[1], $chunkSize);
```

Comment:
What's the change from `audio_buildup` to `audio_bytes` for? Without looking into all the details of the overall implementation I'm wondering how you could change the place where you store the data without changing where you read it?

Reply:
No real reason, just I've neve rliked the audio_buildup term I've used back then so I thought that "bytes" was more simple. 

Re changing where that data can be stored, I really don't know. The HTTP's sysread has to know about it because it should not do a parent's sysread is there are some bytes building up (...). Of course I could add a method to interrogate each "processor" but that would mean method for each of them, or they would need to be a blessed object to call a few methods from. 

That would complicate a fair bit each processors where here it just returns the number of bytes it has build-up everytime it  is called - just one method. I'll continue thinking about that, but don't have a better idea for now.

Reply:
Oh, never mind... I didn't read this correctly (and didn't check whether the value was being used elsewhere).